### PR TITLE
Update Bridge SDK

### PR DESCRIPTION
**Pull request by liujoshua**
* Merge in old reauth to new sessions being written

### CODE REVIEW

**Comment by liujoshua** [comment]
Not ready to merge

File: android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/dao/AccountDAO.java [hunk 1]
```diff
@@ -81,11 +87,33 @@ public void addDataGroup(@NonNull String dataGroup) {
...
+            if (userSessionInfo != null && userSessionInfo.getReauthToken() == null) {
+                UserSessionInfo previous = getUserSessionInfo();
+                if (previous != null) {
+                    userSessionInfo.setReauthToken(previous.getReauthToken());
+                }
+            }
+            
+            setValue(KEY_SESSION_INFO, userSessionInfo, UserSessionInfo.class);
+        } finally {
+            writeLock.unlock();;
```
Comment (dephillipsmichael) [inline android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/dao/AccountDAO.java]: minor - two semi-colons

**Review by dephillipsmichael — APPROVED** [review]
Looks good to me

**Comment by liujoshua** [comment]
Bug with stetho interceptor fixed in rest-client and updated. Good to merge once CI passes

**Comment by dephillipsmichael** [comment]
CI failed

**Pull request merged**